xml/dm_registers.xml

```diff
@@ -37,10 +37,10 @@
             7 (other): An access failed for another reason.
         </field>
 	<field name="allhasreset" bits="19" access="R" reset="-">
-          This field is 1 when all currently selected harts have been reset since the last write to its \Fhasresetack.
+          This field is 1 when all currently selected harts have been reset but not acknowledged.
```

Comment:
I'd go with
"This field is 1 when all currently selected harts have been reset but the reset has not been acknowledged."

Otherwise it's not clear to me what's acknowledged.